### ISSUE TITLE
"ctrl+T" changes the braces position I HATE that

### ISSUE THREAD

**Issue by Master811129** [arduino/arduino-pro-ide#339]
after auto-fomating in Arduino Pro IDE. It changes the braces position (but legacy IDE doesn't):
![image](https://user-images.githubusercontent.com/37829730/92239471-5121cf80-eed0-11ea-9063-e6ae2760321b.png)

**Comment by ubidefeo** [comment]
hi @Master811129 
this is a setting in Clangd format that defaults to such behaviour

the Java IDE does not use ClangD for formatting.
I'm sure this can be tuned in the future, because everyone likes their style.
Arduino sketches are formatted according to a very specific set of rules in Astyle, so I guess that this behaviour will have to be tuned to reflect the one in the Java IDE.
Let's check wtih @per1234 and see what he has to say

**Comment by kittaakos** [comment]
> after auto-fomating in Arduino Pro IDE. It changes the braces position (but legacy IDE doesn't):

Yes, this is a reasonable request. However, we have to figure out how users can/will customize it.
Let's investigate the `BraceWrappingFlags` options from [here](https://clang.llvm.org/docs/ClangFormatStyleOptions.html). [Here](https://zed0.co.uk/clang-format-configurator/) is an online `clangd` formatter with preview.
